pkg/apis/maistra/v2/extension_providers.go

```diff
@@ -57,6 +60,32 @@ type ExtensionProviderEnvoyExternalAuthorizationHTTPConfig struct {
 	// If set, the client request body will be included in the authorization request sent to the authorization service.
 	IncludeRequestBodyInCheck *ExtensionProviderEnvoyExternalAuthorizationRequestBodyConfig `json:"includeRequestBodyInCheck,omitempty"`
 }
+type ExtensionProviderEnvoyExternalAuthorizationGRPCConfig struct {
+	// REQUIRED. Specifies the service that implements the Envoy ext_authz GRPC authorization service.
+	// The format is `[<Namespace>/]<Hostname>`. The specification of `<Namespace>` is required only when it is insufficient
+	// to unambiguously resolve a service in the service registry. The `<Hostname>` is a fully qualified host name of a
+	// service defined by the Kubernetes service or ServiceEntry.
+	//
+	// Example: "my-ext-authz.foo.svc.cluster.local" or "bar/my-ext-authz.example.com".
+	Service string `json:"service"`
+	// REQUIRED. Specifies the port of the service.
+	Port int64 `json:"port"`
+	// The maximum duration that the proxy will wait for a response from the provider (default timeout: 600s).
+	// When this timeout condition is met, the proxy marks the communication to the authorization service as failure.
+	// In this situation, the response sent back to the client will depend on the configured `fail_open` field.
+	Timeout *string `json:"timeout,omitempty"`
+	// Sets a prefix to the value of authorization request header *Path*.
+	// For example, setting this to "/check" for an original user request at path "/admin" will cause the
+	// authorization check request to be sent to the authorization service at the path "/check/admin" instead of "/admin".	FailOpen *bool `json:"failOpen,omitempty"`
+	// Sets the HTTP status that is returned to the client when there is a network error to the authorization service.
+	// The default status is "403" (HTTP Forbidden).
+	FailOpen *bool `json:"fail_open,omitempty"`
```

Comment:
JSON schemas are camel case.
```suggestion
	FailOpen *bool `json:"failOpen,omitempty"`
```